playbooks/openshift/gcp/post-install.yml

```diff
@@ -2,8 +2,6 @@
 
 - hosts: masters[0]
   tasks:
-  - name: configure registry
-    include_role:
-      name:  manage-gcp-infra
-      tasks_from: configure-registry-object-storage.yml
-
+  - name:"Placeholder for GCP Post-install" 
```

Comment:
If there are no post-steps, let's just remove the file... or alternatively just lave it "blank" (only the first `---` left in the file)

Reply:
I think I'll just leave the `---` for now. I think this will still be useful to have in the future should we want to do things like enable and configure other gcp components.